### ISSUE TITLE
Error running example files

### ISSUE THREAD

**Issue by Ida921** [esrud/GONE#48]
Dear Armando,

I encountered this error with example data:

DIVIDE .ped AND .map FILES IN CHROMOSOMES
RUNNING ANALYSIS OF CHROMOSOMES ...
xargs: bash：以状态 255 退出；中止
CHROMOSOME ANALYSES took 55 seconds
Running GONE
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
xargs: bash：以状态 255 退出；中止
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
  Error opening file outfileLD_TEMP/outfileLD_1_GONE_Nebest
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
 END OF ALL PROCESSES
 Specify a valid code for type of genotyping data (0, 1 or 2).
 Specify a valid code for type of genotyping data (0, 1 or 2).
GONE run took 1 seconds
END OF ANALYSES
mv: 无法获取"outfileLD_Ne_estimates" 的文件状态(stat): 没有那个文件或目录
mv: 无法获取"outfileLD_d2_sample" 的文件状态(stat): 没有那个文件或目录
(base) [haosh@node02 Linux]$  Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file  Error opening file outfileLD
outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file  Error opening file outfileLDoutfileLD

 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD
 Error opening file outfileLD Error opening file
 Error opening file outfileLD
 Error opening file outfileLD

Could you please help me to fix this error?

Thank you very much.

Ida

**Comment by Ida921** [comment]
The error is for the example data from your github page

**Comment by Ida921** [comment]
I tried so many times with the example data. But it kept giving me the error above

**Comment by armando-caballero** [comment]
It is an issue with your server, computer system, or similar., as the example has no errors.

**Comment by Ida921** [comment]
You are right. I changed to another server, and it worked out for both example and my own data.
Thank you for you reply.
